MedicalDocManagment.WebUI/app/app.component.ts

```diff
@@ -1,13 +1,17 @@
-﻿/**
- * @fileoverview This file defines AppComponent — root component of front-end side of application.
- * @author Rv-023.Net
- */
-import { Component } from '@angular/core';
+﻿import { Component, ElementRef, OnInit, ViewChild } from '@angular/core';
+
+import { UsersListComponent } from './users-list/users-list.component';
 
 @Component({
   moduleId: module.id,
   selector: 'my-app',
   templateUrl: 'views/app.component.html'
 })
-export class AppComponent {
+export class AppComponent implements OnInit {
+  @ViewChild('usersList') usersList: UsersListComponent;
+
+
+  ngOnInit() {
+    this.usersList.getUsersFromServer();
```

Comment:
Don't you need to subscribe it like so ?
`this.usersList.getUsersFromServer().subscribe(...subscribe's code...);`